src/components/User/Profile.vue

```diff
@@ -113,7 +121,10 @@ p.subtitle
   width 200px
   float right
   max-width: 40%
-
+.on-right
+  margin-left 480px
```

Comment:
This is problematic for smaller screens. You can try out what it looks like on mobile using your browsers Resposive Design Mode (Firefox: https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode)
It should probably be something like "float: right" and maybe some margin (10px or so) to it.